src/wavefunction.lisp

```diff
@@ -330,3 +331,18 @@ If the length/norm of WAVEFUNCTION is known, it can be passed as the LENGTH para
           :do (incf s (probability (aref state i)))
               (setf (aref cdf i) s)
           :finally (return cdf))))
+
+(declaim (inline gaussian-random-complex))
+(defun gaussian-random-complex ()
+  "Return a complex number with Gaussian-random real and imaginary parts."
+  (multiple-value-bind (re im)
+      (alexandria:gaussian-random)
+    (complex re im)))
```

Comment:
`multiple-value-call` imho, but given your expressed wariness of `apply` in other PRs, maybe not.

Reply:
this will compile to more efficient code probably